tests/integration/data/templateForBundleDirectoryTest.json

```diff
@@ -0,0 +1,39 @@
+{
+  "stack": {
+    "name": "mycustomTemplate",
+    "version": "1.1",
+    "os": {
+      "name": "CentOS",
+      "version": "7",
+      "arch": "x86_64",
+      "profile": "Minimal",
+      "pkgs": [
+        {
+          "name": "nginx"
+        }
+      ]
+    },
+    "installation": {
+      "diskSize": 12288
+    },
+    "bundles": [
+      {
+        "name": "myBundle",
+        "version": "6.0",
+        "destination": "/tmp/myBundle",
+        "files": [
```

Comment:
We are using files section to handle a directory. About naming point of view it is a little bit strange. I would like to have @jgweir point of view.